rules.go

```diff
@@ -48,6 +50,8 @@ func _(m fluent.Matcher) {
 		Suggest(`strings.Contains($s1, $s2)`)
 	m.Match(`strings.Count($s1, $s2) == 0`).
 		Suggest(`!strings.Contains($s1, $s2)`)
+	m.Match(`strings.Count($s1, $s2) >= 0`).
+		Report(`statement always true`)
```

Comment:
looks like `bytes.Count` deserve same fame ;}

Reply:
Great catch! Thank you. Let's add it :)